mettle/src/crypttlv.c

```diff
@@ -1,5 +1,21 @@
 #include "crypttlv.h"
 
+#include "mtwister.h"
+#include <unistd.h>
+#include <string.h>
+
+
+void add_weak_encryption(void  *ctr_drbg, void *entropy) {
```

Comment:
I think if we renamed this, it could be a bit more descriptive and future proof incase we need to add others

```suggestion
void add_mtwister_entropy_source(void  *ctr_drbg, void *entropy) {
```

Reply:
It'd also be a good idea to add a log line in here that this is being added as an entropy source. That'd help the reviewer (me) validate that the expected code path is being taken.